ARCH-882 - Fixing some links and adding a ToC to the readme (#4)
im-open/process-code-coverage-summary: README.md

## Limitations 

> GitHub does have a size limitation of 65535 characters for a Status Check body or a PR Comment.  This action will fail if the contents of the summary file exceed the GitHub [limit].

Reviewer: ```suggestion
GitHub has a size limitation of 65535 characters for a Status Check body or a PR Comment.  This action will fail if the contents of the summary file exceed the [limit].
```
